diff --git a/include/urdf_model_state/model_state.h b/include/urdf_model_state/model_state.h
index 0d04e56..fcbd855 100644
--- a/include/urdf_model_state/model_state.h
+++ b/include/urdf_model_state/model_state.h
@@ -37,6 +37,7 @@
 #ifndef URDF_MODEL_STATE_H
 #define URDF_MODEL_STATE_H
 
+#include <cstdint>
 #include <string>
 #include <vector>
 #include <map>
@@ -148,4 +149,3 @@ public:
 }
 
 #endif
-
